web/views.py

```
@@ -1001,7 +1002,51 @@ def mark_session_completed(request, session_id):
 
     messages.success(request, "Session marked as completed!")
     return redirect("course_detail", slug=session.course.slug)
+# start award achievement
 
+from django.shortcuts import render, redirect
```

Ruff (0.8.2)

Redefinition of unused `render` from line 36

Remove definition: `render`

(F811)